### PR TITLE
airdrop wallet balance component

### PR DESCRIPTION
**Pull request by jellyfish-tom**
## What does this PR do?
Introduces wallet balance (header, circle, footer) component.  
image

## Todos:

- [X] Checked on iOS
- [ ] Checked on Android

### CODE REVIEW

File: src/components/AirdropWalletBalanceCircle.tsx [hunk 1]
```diff
@@ -13,14 +13,15 @@ interface Props {
...
-export const WalletBalanceCircle: FC<Props> = ({ balance, threshold }: Props) => (
+export const AirdropWalletBalanceCircle: FC<Props> = ({ balance, threshold }: Props) => (
   <AnimatedCircularProgress
     size={149}
     width={8}
     fill={(balance / threshold) * 100}
     tintColor={palette.textSecondary}
     backgroundColor={palette.lightGrey}
     lineCap="round"
+    style={{ alignSelf: 'center' }}
```
Comment (KedziaPawel) [inline src/components/AirdropWalletBalanceCircle.tsx:24]: add to styles

File: src/components/AirdropWalletBalance.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { AirdropWalletBalanceCircle } from './AirdropWalletBalanceCircle';
+
+interface Props {
+  balance: number;
+  walletName: string;
+  footer: React.ReactNode;
+  threshold: number;
+}
+
+export const AirdropWalletBalance: FC<Props> = ({ balance, walletName, footer, threshold }: Props) => (
```
Comment (KedziaPawel) [inline src/components/AirdropWalletBalance.tsx:15]: :Props not needed

File: src/components/AirdropWalletBalanceCircle.tsx [hunk 3]
```diff
@@ -13,14 +13,15 @@ interface Props {
   threshold: number;
 }
 
-export const WalletBalanceCircle: FC<Props> = ({ balance, threshold }: Props) => (
+export const AirdropWalletBalanceCircle: FC<Props> = ({ balance, threshold }: Props) => (
```
Comment (KedziaPawel) [inline src/components/AirdropWalletBalanceCircle.tsx:16]: :Props not needed

Comment (jellyfish-tom) [reply src/components/AirdropWalletBalanceCircle.tsx:16]: done

File: src/components/AirdropWalletBalance.tsx (hunk 2 above)
Comment (jellyfish-tom) [reply src/components/AirdropWalletBalance.tsx:15]: done

File: src/components/AirdropWalletBalanceCircle.tsx (hunk 1 above)
Comment (jellyfish-tom) [reply src/components/AirdropWalletBalanceCircle.tsx:24]: done

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**